compile.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+rm -rf build
+mkdir build
+cd build
+cmake ..
+make -j8
```

Comment:
I don't think we should add this file to the repo to be honest.

Reply:
Hi, yeah, this file should not be merged, its now in the PR to show that the warnings appear when building with those instructions.